frontend/app/controllers/add-script.js

```diff
@@ -0,0 +1,65 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+
+/*---- Add Script Screen ---*/
+  theme: 'ace/theme/cobalt',
+  themes: [
+    'ace/theme/cobalt',
+    'ace/theme/ambiance',
+    'ace/theme/chaos',
+  ],
+  inputDescription:"",
+  submitFailed: false,
+  failedMessage: "Failed to Login",
+  isProcessing: false,
+  isSlowConnection: false,
+  timeout: null,
+
+  success(response) {
+    this.reset();
+    if (response.status==true){
+      this.reset();
+      window.location.href = "/user-home";
```

Comment:
Instead of this, you should use `this.transitionToRoute('user-home')`

Change all similar occurrences everywhere 

Documentation: https://www.emberjs.com/api/ember/2.14/classes/Ember.Controller/methods/transitionToRoute?anchor=transitionToRoute